utils/remote.cc

```diff
@@ -85,6 +85,7 @@ struct Config
     std::string session_id;
     std::string torrent_ids;
     std::string unix_socket_path;
+    std::string rename_from;
```

Comment:
`Config` is for global parameters that affect all the commands and that need to be accessed outside of `processArgs()`. In this case, if you want to avoid having a global static `rename_from` variable it would be sufficient to declare it as local inside `processArgs()`.